docker/Dockerfile

```diff
@@ -18,6 +18,9 @@ RUN echo 'tzdata tzdata/Areas select America' | debconf-set-selections \
     && rm -rf /var/lib/apt/lists/* \
     && apt clean
 
+# For openbmb/MiniCPM models
+RUN pip3 install datamodel_code_generator
```

Comment:
I think it's better to add this dependency to the existing `RUN` layer.

before
```
RUN python3 -m pip install --upgrade pip setuptools wheel html5lib six \
    && git clone --depth=1 https://github.com/sgl-project/sglang.git \
    && cd sglang \
    && if [ "$BUILD_TYPE" = "srt" ]; then \
         python3 -m pip --no-cache-dir install -e "python[srt]"; \
       else \
         python3 -m pip --no-cache-dir install -e "python[all]"; \
       fi
```

after
```
RUN python3 -m pip install --upgrade pip setuptools wheel html5lib six datamodel_code_generator \
    && git clone --depth=1 https://github.com/sgl-project/sglang.git \
    && cd sglang \
    && if [ "$BUILD_TYPE" = "srt" ]; then \
         python3 -m pip --no-cache-dir install -e "python[srt]"; \
       else \
         python3 -m pip --no-cache-dir install -e "python[all]"; \
       fi
```

Reply:
this one is model specific so I add it with a separate line. We may drop it in the future.

Reply:
ok